tests/integration/vrf/11-multi-vrf-ospf.yml

```diff
@@ -26,6 +26,7 @@ groups:
 vrfs:
   red:
     links: [ dut-r1, dut-r2 ]
+    loopback: True
```

Comment:
I couldn't find where this is being tested? i.e. confirm that dut redistributes this loopback IP via OSPF correctly

Reply:
This effectively tests the configuration template (it should take IP addresses from **loopback_address** not from **loopback**)